testing/test_kerns.py

```diff
@@ -3,7 +3,7 @@
 import tensorflow as tf
 import numpy as np
 import unittest
-from .reference import referenceRbfKernel, referencePeriodicKernel
+from testing.reference import referenceRbfKernel, referencePeriodicKernel
```

Comment:
I thing this was correct as it was? when does the relative import fail? Perhaps what's needed as python2 compatibility using `from __future__ import absolute_import`.